### PR TITLE
{vis}[GCCcore/11.2.0] GTK+ v3.24.31

### PR DESCRIPTION
**Pull request by verdurin**
(created using `eb --new-pr`)

### CODE REVIEW

**Comment by verdurin** [comment]
Test report by @verdurin
**SUCCESS**
Build succeeded for 11 out of 11 (1 easyconfigs in total)
easybuild-c7.novalocal - Linux centos linux 7.9.2009, x86_64, Intel Xeon Processor (Skylake, IBRS), Python 3.6.8
See https://gist.github.com/6132730310ec4b3842b41ba2840190e8 for a full test report.

**Comment by jfgrimm** [comment]
we split GTK+ into GTK2 and GTK3 in #13893

File: easybuild/easyconfigs/g/GTK+/GTK+-3.24.31-GCCcore-11.2.0.eb [hunk 1]
```diff
@@ -0,0 +1,76 @@
+easyblock = 'Bundle'
+
+name = 'GTK+'
```
Comment (jfgrimm) [inline easybuild/easyconfigs/g/GTK+/GTK+-3.24.31-GCCcore-11.2.0.eb:3]:
```suggestion
name = 'GTK3'
```

File: easybuild/easyconfigs/g/GTK+/GTK+-3.24.31-GCCcore-11.2.0.eb [hunk 2]
```diff
@@ -0,0 +1,76 @@
...
+
+default_easyblock = 'ConfigureMake'
+
+default_component_specs = {
+    'sources': [SOURCELOWER_TAR_XZ],
+    'start_dir': '%(namelower)s-%(version)s',
+}
+
+components = [
+    (name, version, {
```
Comment (jfgrimm) [inline easybuild/easyconfigs/g/GTK+/GTK+-3.24.31-GCCcore-11.2.0.eb:43]:
```suggestion
    ('GTK+', version, {
```

Comment (verdurin) [reply easybuild/easyconfigs/g/GTK+/GTK+-3.24.31-GCCcore-11.2.0.eb:43]: Thanks - missed that at the top of my search for `^GTK`...

File: easybuild/easyconfigs/g/GTK3/GTK3-3.24.31-GCCcore-11.2.0.eb [hunk 3]
```diff
@@ -0,0 +1,76 @@
...
+    ('adwaita-icon-theme', '3.38.0', {
+        'preconfigopts': 'autoreconf -f -i && ',
+        'source_urls': [FTPGNOME_SOURCE],
+        'patches': ['adwaita-icon-theme-3.34.3_disable-svg-conversion.patch'],
+        'checksums': [
+            '6683a1aaf2430ccd9ea638dd4bfe1002bc92b412050c3dba20e480f979faaf97',
+            # adwaita-icon-theme-3.34.3_disable-svg-conversion.patch
+            'f4b86855d50759ecfc1e8f6550ec0f3a7a4ea2c80b9f5fc1685fe8967d1c5342',
+        ],
+    }),
```
Comment (lexming) [inline easybuild/easyconfigs/g/GTK3/GTK3-3.24.31-GCCcore-11.2.0.eb:61]: The `adwaita-icon-theme` can be also updated
```suggestion
    ('adwaita-icon-theme', '40.1.1', {
        'preconfigopts': 'autoreconf -f -i && ',
        'source_urls': ['https://ftp.gnome.org/pub/GNOME/sources/%(namelower)s/%(version_major)s'],
        'patches': ['adwaita-icon-theme-3.34.3_disable-svg-conversion.patch'],
        'checksums': [
            '0b6c436ed6ad9887a88ada1f72a0197b1eb73b020d8d344abab4c7fa7250f8f6',
            # adwaita-icon-theme-3.34.3_disable-svg-conversion.patch
            'f4b86855d50759ecfc1e8f6550ec0f3a7a4ea2c80b9f5fc1685fe8967d1c5342',
        ],
    }),
```

**Review by lexming — CHANGES_REQUESTED** [review]

File: easybuild/easyconfigs/g/GTK3/GTK3-3.24.31-GCCcore-11.2.0.eb (hunk 3 above)
Comment (verdurin) [reply easybuild/easyconfigs/g/GTK3/GTK3-3.24.31-GCCcore-11.2.0.eb:61]: @lexming thanks. Do we definitely need to replace the nice `FTPGNOME_SOURCE` macro with an explicit URL?

Comment (lexming) [reply easybuild/easyconfigs/g/GTK3/GTK3-3.24.31-GCCcore-11.2.0.eb:61]: Unfortunately yes 😞, they changed the structure of the URL for the new versions of adwaita

Comment (verdurin) [reply easybuild/easyconfigs/g/GTK3/GTK3-3.24.31-GCCcore-11.2.0.eb:61]: Sigh. Thanks for picking that up.

**Comment by verdurin** [comment]
Test report by @verdurin
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in total)
centos7.lan - Linux centos linux 7.9.2009, x86_64, Intel Core Processor (Skylake, IBRS), Python 3.6.8
See https://gist.github.com/4bab27c9906a9ea40e0edcc401c81b44 for a full test report.

**Comment by lexming** [comment]
Going in, thanks @verdurin !

**Pull request merged**